app/models/project_group.rb

```diff
@@ -62,6 +62,11 @@ def summary_or_none
     summary.blank? ? "[#{I18n.t("none")}]" : summary.to_s
   end
 
+  # For the "Group" dropdown in step edit form
```

Comment:
I'd prefer to avoid comments that specifically name pieces of the interface because later if something else uses this (likely) then the comment becomes incomplete. Can you please change to something more generic. Also the method name could perhaps be `indented_option_label` to be a bit more clear about the function.